### DISCUSSION TITLE
Upgrading python@3.9 3.9.1_8 -> 3.9.4

### DISCUSSION THREAD

**Discussion by duemme** [Homebrew/discussions#1322] [Getting started]
I get this error while trying to upgrade:

Error: python@3.9: the bottle needs the Apple Command Line Tools to be installed.
  You can install them, if desired, with:
    xcode-select --install

I'm running Big Sure 11.2.3.

Do I have to install xcode via homebrew?

**Comment by SMillerDev** [comment]
You don't need xcode, the bottle/software needs the Apple Command Line Tools to be installed.

You can install them, if desired, with:
`xcode-select --install`

**Comment by carlocab** [comment]
You need to have the Xcode Command Line Tools to install `python@3.9` from a bottle. If you don't want to install the CLT, but have Xcode.app installed, you can try to build `python@3.9` from source. (Note that building from source is unsupported.)

To install the Xcode CLT, run
```
xcode-select --install
```
Not sure what you mean by your question, as `brew` does not install Xcode.

**Comment by duemme** [comment]
Thanks. Now it is working. I thought I had to install xcode via homebrew. Sorry
